src/objects/Opportunity.object

```diff
@@ -832,6 +832,19 @@
         <trackTrending>false</trackTrending>
         <type>Lookup</type>
     </fields>
+    <fields>
+        <fullName>Batch_Number__c</fullName>
+        <description>The batch that created this Opportunity. This field is automatically populated based on your Batch Number Settings in NPSP Settings.</description>
+        <externalId>false</externalId>
+        <inlineHelpText>The batch that created this Opportunity. This field is automatically populated based on your Batch Number Settings in NPSP Settings.The batch that created this Opportunity.</inlineHelpText>
```

Comment:
```suggestion
        <inlineHelpText>The batch that created this Opportunity. This field is automatically populated based on your Batch Number Settings in NPSP Settings.</inlineHelpText>
```

Reply:
@dfuller114 looks like some of the text ended up getting duplicated. I just removed the additional sentence.

Reply:
@melissabarber ah cool. didn't notice that. just updated!